### DISCUSSION TITLE
Welcome to PISM Discussions!

### DISCUSSION THREAD

**Discussion by ckhroulev** [pism/pism#468] [General]
## Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by talbrecht** [comment]
Hej PISM community, 
I am involved in this community since 2008 or so and have been working mostly on the Antarctic Ice Sheet, with focus on ice shelf calving, fracture evolution (continuum damage mechanics) and sub-shelf melting (PICO). I have run ensembles of glacial cycle simulations and got recently involved in the coupling to the ocean (FESOM) and to the solid-Earth (incl. gravitationally consistent sea level, VILMA). Find out more on my [PIK website](https://www.pik-potsdam.de/members/albrecht). 
Best, 
Torsten

**Comment by citibeth** [comment]
Hello, I've been working with PISM since 2012, coupling it with the GISS GCM climate model.  Now I'm at UAF, using PISM as part of investigations of tidewater glaciers, calving and ice front melting in Greenland.

-- Elizabeth

**Comment by emily1865** [comment]
Hi! My name is Emily and I have worked with PISM previously during my master at University of Copenhagen, for a paleo modeling project in Greenland. I am currently attempting to implement it for paleo modeling (and hopefully isochrone tracking) in Antarctica at University of Washington, for my PhD. Good to meet you all! :)

**Reply by ckhroulev** [reply]
Hi Emily! Please don't hesitate to reach out if you have any questions!
